src/executor/query.rs

```diff
@@ -19,6 +19,13 @@ pub(crate) enum QueryResultRow {
     SqlxSqlite(sqlx::sqlite::SqliteRow),
     #[cfg(feature = "mock")]
     Mock(crate::MockRow),
+    Disconnected,
```

Comment:
And QueryResultRow too

Reply:
Same here.

> I originally want to have a placeholder to avoid the use of
> 
> ```rust
> #[allow(unreachable_patterns)]
> _ => unreachable!(),
> ```